pkg/apis/controller/v1beta1/appwrapper.go

```diff
@@ -263,6 +263,14 @@ type AppWrapperStatus struct {
 
 	// Represents the latest available observations of pods under appwrapper
 	PendingPodConditions []PendingPodSpec `json:"pendingpodconditions"`
+
+	//Resources consumed
+
+	TotalCPU float64 `json:"totalcpu,omitempty"`
+
+	TotalMemory float64 `json:"totalmemory,omitempty"`
+
+	TotalGPU int64 `json:"totalgpu,omitempty"`
```

Comment:
Please add the fields to the CRD(S). The missing fields do throw warnings when upgrading to k8s, and in the tests created by RH for the Codeflare operator.

Reply:
Thanks, Done